src/server/routes/relayer/index.ts

```diff
@@ -30,6 +31,7 @@ const BodySchema = Type.Union([
       gas: Type.String(),
       nonce: Type.String(),
       data: Type.String(),
+      chainid: Type.Optional(Type.String()),
```

Comment:
I guess `Chain-ID` has to be all lower case & not camel-case, as the SDK has `chainid` in the request body and the contract ABI verify function signature too has `chainid`?

Reply:
Yeah, this was a bug caught during testing. It seems to match the evm block.chainid convention. I would like to make it camelcase, but think it would be confusing to users if we had cc interface, but lowercase when calling the contract. Also, they expect the request message params to be directly passed